src/core/network.service.ts

```diff
@@ -273,6 +273,7 @@ export class NetworkService implements OnApplicationBootstrap {
       GetIndexerUnfinalisedPlansQuery,
       GetIndexerUnfinalisedPlansQueryVariables
     >({
+      // @ts-ignore
```

Comment:
this is due to graphql version conflict, upgrade it to `"graphql": "^16.7.1"` can fix the problem